README.md

```diff
@@ -6,25 +6,37 @@ Local coordinator for offline development of Hathora applications.
 
 ## How to use
 
-### Setup
+### 1. Enable HTTPS for localhost
 
-Generate certs using mkcert:
+First, generate certificates using [mkcert](https://github.com/FiloSottile/mkcert).
+
+In your project root directory:
 
 ```sh
 mkcert -install
 mkcert localhost
 ```
 
-Install dependencies:
+Make sure to add `*.pem` to your `.gitignore` file to avoid committing the certificate files.
 
-```sh
-npm install
-```
+### 2. Add `local-coordinator` to your Hathora project
 
-### Run
+In your project root directory:
 
-Start the coordinator process:
+- Install with `npm i https://github.com/hathora/local-coordinator.git`
```

Comment:
what's the advantage of doing this vs just running `npm start` in this repo?

i guess you get to avoid having to clone this repo?

what about suggesting that people globally install this package? but not sure how they would install the cert files to it

Reply:
i don't think you would normally have cert files in your hathora project.

what's the main pain point you currently have with the local-coordinator with hathora?

Reply:
The main pain point is having two separate directories - one for the game and another for local-coordinator

However, there are many ways to automate this.

I'm open to close this and focus on more important improvements.